src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -53,34 +60,66 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
                     resource_group_name=None,
                     target=None,
                     auth_mode=None):
+
+    registry, resource_group_name = get_registry_by_name(
+        cmd.cli_ctx, registry_name, resource_group_name)
+
+    if context_path.lower() == NULL_CONTEXT:
+        context_path = None
+        commit_trigger_enabled = False
+        pull_request_trigger_enabled = False
+
     if (commit_trigger_enabled or pull_request_trigger_enabled) and not git_access_token:
         raise CLIError("If source control trigger is enabled [--commit-trigger-enabled] or "
                        "[--pull-request-trigger-enabled] --git-access-token must be provided.")
 
-    if file.endswith(ALLOWED_TASK_FILE_TYPES):
-        FileTaskStep = cmd.get_models('FileTaskStep')
-        step = FileTaskStep(
-            task_file_path=file,
-            values_file_path=values,
+    yaml_template = ""
+    if cmd_value:
+        yaml_template = "steps: \n  - cmd: {{ .Values.command }}\n"
+        values_content = "command: {0}\n".format(cmd_value)
+
+    if file == "-":
```

Comment:
if user pass --cmd and -f at the same time, we should block and the error message should tell users they can only pass --cmd or -f in the command.

Reply:
For now, file has the default value 'acb.yaml'. There is no way to tell if -f and --cmd are passed at the same time once inside the acr_task_create or acr_run.

Reply:
Unless file removes its default value from 'acb.yaml' to None.

Reply:
Are we going to support a list of `--cmd` and a list of `--timeout` for `az acr run` and `az acr task create`?